plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/SwtBotAppEngineActions.java

```diff
@@ -46,15 +48,20 @@
    * @return the project
    */
   public static IProject createNativeWebAppProject(SWTWorkbenchBot bot, String projectName,
-      String location, String javaPackage) {
-    return createWebAppProject(bot, projectName, location, javaPackage, null /* extraBotActions */);
+      String location, String javaPackage, AppEngineRuntime runtime) {
+    return createWebAppProject(bot, projectName, location, javaPackage, runtime,
+        null /* extraBotActions */);
   }
 
-  /** Create a new project with the Maven-based Google App Engine Standard Java Project wizard */
+  /**
+   * Create a new project with the Maven-based Google App Engine Standard Java Project wizard
+   * 
+   * @param runtime
+   */
   public static IProject createMavenWebAppProject(final SWTWorkbenchBot bot, String projectName,
-      String location, String javaPackage,
-      final String mavenGroupId, final String mavenArtifactId) {
-    return createWebAppProject(bot, projectName, location, javaPackage, new Runnable() {
+      String location, String javaPackage, AppEngineRuntime runtime, final String mavenGroupId,
```

Comment:
This is used by existing tests; I've just added an `AppEngineRuntime` parameter.